.gitignore

```diff
@@ -19,6 +19,9 @@ bin/
 *.backup
 *.debug
 *.dump
+out/
+.java-version
```

Comment:
Also, what created this `.java-version` file?

It would be better to identify the origin and properly configure the tooling to not create those.
Or, if its a testcase, make use of the target/ directory properly.